pysher/pusher.py

```diff
@@ -12,16 +12,21 @@ class Pusher(object):
     client_id = 'PythonPusherClient'
     protocol = 6
 
-    def __init__(self, key, secure=True, secret=None, user_data=None, log_level=logging.INFO,
+    def __init__(self, key, cluster=None, secure=True, secret=None, user_data=None, log_level=logging.INFO,
                  daemon=True, port=None, reconnect_interval=10, custom_host=None, auto_sub=False,
                  http_proxy_host=None, http_proxy_port=None, http_no_proxy=None, http_proxy_auth=None,
                  **thread_kwargs):
+        # https://pusher.com/docs/clusters
+        if cluster:
+            self.host = f"ws-{cluster}.pusher.com"
```

Comment:
literal string interpolation is 3.6+, but pysher should be backwards compatible up to version 3.5. 
Please use `format()` instead. Cheers!